src/styles/components/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  width: $calendar-width;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  transition: 0.2s ease-in-out;
+
+  // adding correct margin-left to needed ceil of day
```

Comment:
- Your calendar doesn't have any effects, only a `calendar __day` 😉 
- get rid of all comments

Reply:
I fixed:

- rid of transition on 7 line and fix transition on 35 line.
- seems to fix nesting in 'for' & 'each'. (i not good with 'for' and 'each')
- rid of comments. (it's sad cause maybe in future i'm returns to this sample)
- fix blank line. (don't understand why linter catch error without this space, but now that fixed)